docs/src/performance.md

```diff
@@ -0,0 +1,4 @@
+# Performance Tips
```

Comment:
This isn't included anywhere. 

I guess we could move the 'a note on iteration' from the API page here though?

Reply:
This file is still here, but not linked anywhere...